plugins/catalog-graphql/src/schema.gql

```diff
@@ -0,0 +1,25 @@
+type EntityMetadataAnnotation {
+  key: String!
+  value: String!
+}
+
+type EntityMetadata {
+  name: String!
+  namespace: String
+  annotation(name: String): EntityMetadataAnnotation
+  annotations: [EntityMetadataAnnotation!]
```

Comment:
all lists should be non-null (`!`) because of the better consumer experience, so maybe add a resolver for `annotations` that translates to the empty list if it's missing

Reply:
and maybe add `labels` too while we're at it? and `uid`, `etag`, `generation`?